lms/djangoapps/courseware/tests/test_views.py

```diff
@@ -809,7 +849,8 @@ def set_up_course(self, **course_kwargs):
         """
         course = CourseFactory.create(**course_kwargs)
         chapter = ItemFactory.create(category='chapter', parent_location=course.location)
-        section = ItemFactory.create(category='sequential', parent_location=chapter.location, due=datetime(2013, 9, 18, 11, 30, 00))
+        section = ItemFactory.create(category='sequential', parent_location=chapter.location,
+                                     due=datetime(2013, 9, 18, 11, 30, 00))
```

Comment:
Nit: Put all arguments on one line or each argument on a separate line, indented four spaces, with the final right paren on its own line.